Sample/MainActivity.cs

```diff
@@ -37,6 +32,7 @@
 using PDFTextStripper = Com.Tom_roush.Pdfbox.Text.PDFTextStripper;
 using PDFBoxResourceLoader = Com.Tom_roush.Pdfbox.Util.PDFBoxResourceLoader;
 using ImageType = Com.Tom_roush.Pdfbox.Rendering.ImageType;
+using AndroidX.AppCompat.App;
```

Comment:
if you like you can move this below
using Android.Widget;

Reply:
Done 👍